examples/flutter_timer/lib/bloc/timer_event.dart

```diff
@@ -7,25 +7,25 @@ abstract class TimerEvent extends Equatable {
   List<Object> get props => [];
 }
 
-class Start extends TimerEvent {
+class TimerRunStarted extends TimerEvent {
```

Comment:
```suggestion
class TimerStarted extends TimerEvent {
```
Thoughts?

Reply:
Well, here I was also confused a bit between `TimerStarted` and `TimerRunStarted` as the Noun in between is optional, but I decided to go with the complete anatomy even though `TimerStarted` made a bit more sense.
I think we can remove Run from all timer events and still follow convention.